Pollo/Views/PollBuilder/OnboardingView.swift

```diff
@@ -0,0 +1,524 @@
+        }
+        
+        secondCorrectView.snp.makeConstraints { make in
+            make.width.height.equalTo(correctViewLength)
+            make.centerY.equalTo(secondOptionView)
+            make.leading.equalTo(secondOptionView).offset(correctViewLeadingOffset)
+        }
+        
+        secondOptionLabel.snp.makeConstraints { make in
+            make.centerY.equalTo(secondCorrectView)
+            make.leading.equalTo(secondCorrectView.snp.trailing).offset(optionLabelLeadingOffset)
+        }
+        
+        addOptionView.snp.makeConstraints { make in
+            make.leading.trailing.equalTo(firstOptionView)
+            make.height.equalTo(53)
+            make.top.equalTo(secondOptionView.snp.bottom).offset(addOptionViewTopOffset)
+        }
+        
+        plusLabel.snp.makeConstraints { make in
+            make.width.equalTo(plusLabelWidth)
+            make.leading.equalTo(addOptionView).offset(optionHorizontalPadding)
+            make.centerY.equalTo(addOptionView)
+        }
+        
+        addOptionLabel.snp.makeConstraints { make in
+            make.leading.equalTo(plusLabel.snp.trailing).offset(addOptionLabelLeadingOffset)
+            make.centerY.equalTo(addOptionView)
+        }
+        
+        animatedLabel.snp.makeConstraints { make in
+            make.top.equalTo(addOptionView.snp.bottom).offset(animatedLabelTopOffset)
+            make.leading.trailing.equalTo(addOptionView)
+        }
+        
+        buttonView.snp.makeConstraints { make in
+            make.width.equalTo(buttonViewWidth)
+            make.height.equalTo(buttonViewHeight)
+            make.bottom.equalTo(safeAreaLayoutGuide.snp.bottom).offset(-buttonViewBottomOffset)
+            make.trailing.equalToSuperview().inset(buttonViewHorizontalPadding)
+        }
+        
+        buttonLabel.snp.makeConstraints { make in
+            make.center.equalTo(buttonView)
+        }
+
+        transitionButton.snp.makeConstraints { make in
+            make.edges.equalToSuperview()
+        }
+    }
+    
+    func getAnimatedLabelLeadingOffset() -> CGFloat {
+        if stage == .autofillChoices {
+            return 89
+        } else if stage == .quizMode {
+            return 18
+        }
+        return 177
+    }
+    
+    func getAnimatedLabelTrailingInset() -> CGFloat {
+        if stage == .autofillChoices {
+            return 16
+        } else if stage == .quizMode {
+            return 92
+        } else if stage == .saveDraft {
+            return 130
+        }
+        return 20
+    }
+    
+    @objc func transitionButtonTapped() {
+        if !shouldTransition { return }
+        shouldTransition = false
+        switch stage {
+        case .welcome:
+            stage = .createQuestion
+            animatedLabel.text = getAnimatedLabelText()
+            UIView.animate(withDuration: animationDuration, animations: {
+                self.welcomeContainerView.alpha = 0
```

Comment:
I'm curious if it would be easier if you combined all of the views into superviews for each "slide" of the onboarding flow. This would avoid having to individually change the alpha values for each label/view within the slide.

Reply:
Great idea! I'll try that right now.

Reply:
@jackthmp This is done for `welcomeContainerView` for the first slide, but I'm not able to combine the rest of the views into superviews because they are reused throughout the remaining slides.

Reply:
Ahh I see, dropping this then